diff -Nur -x .git rtmpdump/librtmp/rtmp_sys.h rtmpdump-o/librtmp/rtmp_sys.h
--- rtmpdump/librtmp/rtmp_sys.h	2019-10-27 20:40:13.290886660 +0800
+++ rtmpdump-o/librtmp/rtmp_sys.h	2019-10-27 20:39:21.227721537 +0800
@@ -50,6 +50,7 @@
 #include <netinet/in.h>
 #include <netinet/tcp.h>
 #include <arpa/inet.h>
+#include <limits.h>
 #define GetSockError()	errno
 #define SetSockError(e)	errno = e
 #undef closesocket
diff -Nur -x .git rtmpdump/rtmpdump.c rtmpdump-o/rtmpdump.c
--- rtmpdump/rtmpdump.c	2019-10-27 20:40:13.290886660 +0800
+++ rtmpdump-o/rtmpdump.c	2019-10-27 20:39:21.231723552 +0800
@@ -67,7 +67,7 @@
 #endif
 }
 
-inline void
+static inline void
 CleanupSockets()
 {
 #ifdef WIN32
